container/sagemaker/tensorflow-serving.js

```diff
@@ -14,11 +14,30 @@ function invocations(r) {
 }
 
 function ping(r) {
-    // TODO replace with call to Model Status API when Tensorflow Serving 1.12 is available
+    if ('1.11' == r.variables.tfs_version) {
+        return ping_tfs_1_11(r)
+    }
+
+    var uri = remove_tfs_method(make_tfs_uri(r))
```

Comment:
I would modify make_tfs_uri to handle this case. YMMV.

Reply:
updated

Reply:
tests for all the js behavior are done in container in test/integration/test_container.py